bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/select/SelectJSReference.java

```diff
@@ -26,6 +26,6 @@ public static SelectJSReference instance() {
      * Private constructor.
      */
     private SelectJSReference() {
-        super(SelectJSReference.class, "js/bootstrap-select.js");
+        super("bootstrap-select/current/dist/js/bootstrap-select.js");
```

Comment:
Isn't there a `.min.js` in the webjar ?
Asking because you used the `min` version in SelectCSSReference.java